library/src/shared/services/account/account.service.ts

```diff
@@ -0,0 +1,161 @@
+import { Injectable, OnDestroy } from '@angular/core';
+
+import { forkJoin, Observable, Subject, map, catchError, of } from 'rxjs';
+
+// Client
+import {
+  AccountBankModel,
+  AccountsService,
+  AssetBankModel,
+  PricesService,
+  SymbolPriceBankModel
+} from '@cybrid/cybrid-api-bank-angular';
+
+// Services
+import { AssetService, Asset } from '@services';
+
+// Utility
+import { symbolSplit } from '@utility';
+import { AssetPipe } from '@pipes';
+
+export interface Account {
+  asset: Asset;
+  counter_asset: Asset;
+  price: SymbolPriceBankModel;
+  value: number;
+  account: AccountBankModel;
+}
+
+export interface AccountOverview {
+  accounts: Account[];
+  balance: number;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AccountService implements OnDestroy {
+  private unsubscribe$ = new Subject();
+
+  constructor(
+    private accountsService: AccountsService,
+    private pricesService: PricesService,
+    private assetService: AssetService,
+    private assetPipe: AssetPipe
+  ) {}
+
+  ngOnDestroy() {
+    this.unsubscribe$.next('');
+    this.unsubscribe$.complete();
+  }
+
+  // Filter for 'trading' accounts
+  filterAccounts(): Observable<AccountBankModel[]> {
+    return this.accountsService.listAccounts().pipe(
```

Comment:
Does accountsService.listAccounts() take care of pagination, i.e., will it list all the accounts regardless of how many pages of accounts there are?